src/frontend/src/handler/create_table.rs

```diff
@@ -264,10 +264,20 @@ pub async fn handle_create_table(
     table_name: ObjectName,
     columns: Vec<ColumnDef>,
     constraints: Vec<TableConstraint>,
+    if_not_exists: bool,
 ) -> Result<RwPgResponse> {
     let session = context.session_ctx.clone();
 
-    session.check_relation_name_duplicated(table_name.clone())?;
+    if let Err(e) = session.check_relation_name_duplicated(table_name.clone()) {
```

Comment:
I think there is still a small chance that this doesn't work when two people create same table name together.

Reply:
Yes, I am aware of that since we release the lock after the check. But since we have the same implementation in create database/schema/indez, I would rather keep the behaviour consistent in this PR.

Reply:
> I think there is still a small chance that this doesn't work when two people create same table name together.

Even if we could check it in one frontend, the problem would still exist when in a multi frontend environment. The only way to ensure consistency is to rely on the meta service, and perhaps we still need some refine based on the errors returned by the meta: when `if_not_exists` set and meta returns duplicate error, frontend should return ok rather than the original error. Cc @HuaHuaY